src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -624,13 +624,8 @@ private static (uint userId, uint groupId) GetUserAndGroupIds(ProcessStartInfo s
         private unsafe static (uint? userId, uint? groupId) GetUserAndGroupIds(string userName)
         {
             Interop.Sys.Passwd? passwd;
```

Comment:
maybe worth a ocmment like // note on Centos, GetPwNamR does not give back an accurate required buffer size, so size generously

Reply:
done